next/components/forms/SpaceReservationForm/index.tsx

```diff
@@ -300,17 +300,23 @@ const SpaceReservationForm = () => {
           <div>
             {/* hudobne studio */}
             <iframe
+              title="Hudobne studio"
```

Comment:
Shouldn't the title also be translated?

Reply:
title is not visible, but i guess it's goot for accesibility to have it - additionaly added  in commit